deploy-dotorg.sh

```diff
@@ -19,10 +19,10 @@ declare -a THEMES_TO_DEPLOY=(
 	"blockbase" 
 	"zoologist"
 	"geologist"
-	"mayland-blocks"
```

Comment:
Actually looks like I was wrong about mayland blocks, it's here: https://wordpress.org/themes/mayland-blocks/